src/tree_types.ml

```diff
@@ -11,10 +11,8 @@ module TC =
     type t =
       | Symb of int * string * Files.module_path
```

Comment:
Would you mind to put those arguments in reverse order, `Symb of Files.module_path * string * int`, or does this imply too many changes?

Reply:
But this order allow the pervasive comparison to start comparing the int, which should be faster (although one can argue that we should favour code readability)

Reply:
Are you sure that OCaml compares the arguments from left to right? If you want to be sure of that, it seems better to define your own comparison function.

Reply:
Indeed, I'll change this